### PR TITLE
reordering links from conceptual, to dev tools, then actual code

### PR DESCRIPTION
**Pull request by benjguin**

### CODE REVIEW

**Review by jchomarat — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,9 +13,8 @@ Here are some links to these sharing artifacts:
 
 - [Dev Approach Data Projects](https://ssrikantan.github.io/blog/2021/04/08/dev-approach-data-projects)
 - [Why you might want to have an end to end walkthrough document in your repo](https://blog.3-4.fr/2021/04/12/end-to-end-walkthrough/)
-- [Python logger class for Azure using Opencensus Python SDK](https://github.com/anandchugh/python_logger_opencensus_azure)
```
Comment (ankitbko) [inline README.md:16]: Why is this removed?

Comment (benjguin) [reply README.md:16]: it's not removed. It's moved to the end so that we have:
- a better way to logging in Python
- then, the logging class that applies, amongst other things what you recommended

I think it's better than
- a logging class
- a better way to logging

;-)

Please resolve conversation if you're OK

Comment (ankitbko) [reply README.md:16]: ah ok missed seeing it

**Review by ankitbko — APPROVED** [review]
